thirdeye/thirdeye-frontend/app/pods/rootcause/template.hbs

```diff
@@ -77,7 +77,10 @@
         </div>
         <div class="row rootcause-compare">
           {{rootcause-select-comparison-range
-            range=context.anomalyRange
+            maxTime=context.maxTime
```

Comment:
instead of the metric max time, set the max time to the EOD today. Multiple metrics can have different maxTime values, so there isn't a "true" maxTime anyways.

Reply:
Yeah it was set to `maxDisplayWindow` but i want to be precise and have it not select future time that the current baseline does not provide. So i try to set it to the current baseline's `max time`. Let me set it back to EOD.

Reply:
actually yes, you're right. the max time should be analysisRange[1]

Reply:
good catch. I updated to do the max display window range (analysisRange[1]). Thanks.